doc/source/dev/roadmap-detailed.rst

```diff
@@ -75,15 +67,6 @@ NumPy C API (by Cython, everything in SciPy was fixed) will disappear. Then we
 can define ``NPY_NO_DEPRECATED_API`` unconditionally.
```

Comment:
Actually this also happened. Not relevant but just saying.

Reply:
I think 99% there, but it's only fully true if we drop Cython 2.x and then update our use of `numpy_nodepr_api`.